Riot/Modules/Room/RoomCoordinator.swift

```diff
@@ -33,6 +33,7 @@ final class RoomCoordinator: NSObject, RoomCoordinatorProtocol {
     private var selectedEventId: String?
     private var loadingCancel: UserIndicatorCancel?
     private var locationSharingIndicatorCancel: UserIndicatorCancel? // Used for location sharing advertizements
+    private var foundDiscussionTargetUser: MXUser?
```

Comment:
@yostyle can you please explain why we need this private var?
Are we calling several times `loadRoom(withDiscussionTargetUserId...`?

Reply:
please remove this private foundDiscussionTargetUser